packages/core/admin/admin/src/content-manager/pages/ListView/components/BulkActionButtons/SelectedEntriesModal/index.js

```diff
@@ -197,12 +264,49 @@ const BoldChunk = (chunks) => <Typography fontWeight="bold">{chunks}</Typography
 const SelectedEntriesModalContent = ({ onToggle, onConfirm, onRefresh }) => {
   const { formatMessage } = useIntl();
   const { selectedEntries, rows, isLoading, isFetching } = useTableContext();
+  const [isDialogOpen, setIsDialogOpen] = React.useState(false);
+  const [isConfirmButtonLoading, setIsConfirmButtonLoading] = React.useState(false);
+  const [rowsToDisplay, setRowsToDisplay] = React.useState([]);
+  const [entriesToPublish, setEntriesToPublish] = React.useState([]);
```

Comment:
Can these not be derived by using `rows` and `selectedEntries`?

Reply:
I was able to derive entriesToPublish, but rows was a little more complicated since I don't want it to update on every render but just during the publish action.

Reply:
I knew there was something to be done, thanks for exploring.